modulefiles/module_base.orion

```diff
@@ -1,15 +1,16 @@
-#%Module######################################################################
-##
-##      FV3GFS prerequisites
-##
+#%Module################################################
+# FV3GFS prerequisites
+########################################################
 
 module use /apps/contrib/NCEP/libs/hpc-stack/modulefiles/stack
 module load hpc/1.1.0
 module load hpc-intel/2018.4
+
+# gsi
+module unload mkl/2020.2
+module load mkl/2018.4
```

Comment:
why is this necessary here?  The GSI modulefile at build time does this switch.
Can you explain why/where this is used at runtime?

Reply:
Will contact GSI team for guidance.

Reply:
Background information is provided in NOAA-EMC/GSI issue [#110](https://github.com/NOAA-EMC/GSI/issues/110) and NOAA-EMC/hpc-stack issue [#170](https://github.com/NOAA-EMC/hpc-stack/issues/170).   @MichaelLueken-NOAA added to keep him in the loop.